README.md

```diff
@@ -1,26 +1,82 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+//while loop
+let num = 20
+while (num >= 2) {
+  console.log(num)
+  num -= 1
+}
+
+
+//for loop
+for (let num = 20; num > 1; num -= 1) {
+  console.log(num)
+}
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+//while loop
+let num = 10;
+while(num >= 0) {
+ if(num % 2 === 0) {
+   console.log(num+ ' is even number')
+ } else {
+   console.log(num + ' is odd number')
+ }
+ num--
+}
+
+//for loop
+let num = 17
+for (let i = 0; i <= num; i++){
+  if (i % 2 === 0){
+    console.log(i + ' is even')
+  } else {
+    console.log(i + ' is odd')
+  }
+}
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
-  <details>
-    <summary>
-      Hint
-    </summary>
-    Find the final number and increment the loop by 9.
-  </details>
-  
-4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+let num = 10
+while (num >= 0) {
+  console.log(num+ ' * 9 = ' +num*9)
+  num --
+}
+
+//_Bonus_ think of another way to solve it.
+// newp.
+//  <details>
+//    <summary>
+//      Hint
+//    </summary>
+//    Find the final number and increment the loop by 9.
+//  
+
+4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers
+
 
+for (let i=1; i < 101; i++){
+    if (i % 5 == 0) console.log("Buzz");
```

Comment:
Watch out here - for 15, for example, we'd log "Buzz", when it's possible that we wanted "Fizz." We didn't specify, but I wanted to make sure you knew.